.github/workflows/maven-build.yml

```diff
@@ -15,23 +15,42 @@ jobs:
       GITHUB_TOKEN: ${{ github.token }}
       MARKDOWN2HTML_MAVEN_PLUGIN_FAIL_ON_ERROR: true
     steps:
-      - name: Checkout
+      # Checkout the repository
```

Comment:
why do you duplicate everything from "- name:"  to comments
please remove the comments and add missing info "- name:"

Reply:
comments are required to have an empty line, otherwise it will be yaml without spaces